mlkem/native/aarch64/intt_clean.S

```diff
@@ -226,11 +234,41 @@ _intt_asm_clean:
         ASM_LOAD(xtmp, const_addr)
         ld1 {consts.8h}, [xtmp]
 
+        ninv             .req v29
+        ninv_tw          .req v30
+
+        ASM_LOAD(xtmp, ninv_addr)
+        ld1r {ninv.8h}, [xtmp]
+        ASM_LOAD(xtmp, ninv_tw_addr)
+        ld1r {ninv_tw.8h}, [xtmp]
+
+        mov inp, in
+        mov count, #8
+
+scale_start:
```

Comment:
What's going on here? This doesn't seem to be much to do with "documenting the bounds"

Reply:
Please see the commit messages -- we're hoisting the scaling in the invNTT from end to start, as a means to simultaneously scale and reduce the size of the representatives.